tests/puppeteer/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/general/01_enableFinalSummary.js

```diff
@@ -106,8 +112,14 @@ describe('Enable final summary', async () => {
       // Check the final summary existence in payment step
       const isVisible = await this.pageObjects.orderConfirmationPage.isFinalSummaryVisible();
       await expect(isVisible).to.be.equal(test.args.exist);
+    });
+
+    it('should go back to BO', async function () {
```

Comment:
Could it be more informative ? why do you go back to the BO exactly ?

Reply:
Its a foreach : 

- Change setting
- Go to FO
- Verify the setting
- Go back to BO

And Go to Step 1